Products/zms/conf/metaobj_manager/com.zms.foundation/ZMSTable/interface0.zpt

```diff
@@ -354,7 +360,7 @@
 							xml += '<item key="colspan" type="int">' + 1 + '</item>\n';
 							var el_inputs = $('textarea,input',el_tds[j]);
 							for ( var k=0; k < el_inputs.length; k++) {
-								var key = el_inputs[k].id.substring(0,el_inputs[k].id.indexOf('_'));
+								var key = el_inputs[k].id.substr(0,el_inputs[k].id.indexOf('_'));
```

Comment:
substr() is Deprecated: This feature is no longer recommended.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr?retiredLocale=de